source/routing/raptor.cpp

```diff
@@ -705,11 +808,44 @@ void RAPTOR::raptor_loop(Visitor visitor, const nt::RTLevel rt_level, uint32_t m
                     // We try to get on a vehicle, if we were already on a vehicle, but we arrived
                     // before on the previous via a connection, we try to catch a vehicle leaving this
                     // journey pattern point before
+
+                    // if we cannot board at this stop point, nothing to do
+                    if (!prec_labels.transfer_is_initialized(jpp.sp_idx) || !valid_stop_points[jpp.sp_idx.val]) {
+                        continue;
```

Comment:
If `is_onboard==true`, shouldn't we do the stay_in check https://github.com/CanalTP/navitia/pull/2937/files#diff-30631442826dc5f74f14e880e96c74bcR892 like before or is it pointless ?